org.syphon.Syphon.json

```diff
@@ -0,0 +1,97 @@
+{
+    "app-id": "org.syphon.Syphon",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "41",
+    "sdk": "org.gnome.Sdk",
+    "command": "syphon",
+    "separate-locales": false,
+    "finish-args": [
+        "--share=ipc",
+        "--socket=x11",
+        "--socket=fallback-x11",
+        "--socket=wayland",
+        "--socket=pulseaudio",
+        "--share=network",
+        "--device=all",
```

Comment:
why do you need "device all"

Reply:
Syphon currently supports sending photos from the webcam and support for video/calls and voice messages is planned. I believe this is required to access the webcam and mic, correct?

Reply:
webcam. fair enough.